src/main/resources/dbmigration/eureka-ontology-EK_DEMOG-changelog.xml

```diff
@@ -2084,20 +2084,47 @@
             <column name="C_SYMBOL" value="DEM|SEX:Unspecified"/>
             <column name="EK_UNIQUE_ID" value="DEM|SEX:Unspecified"/>
         </insert>
-        <insert tableName="EK_DEMOG">
+        <insert tableName="EK_DEMOG" dbms="h2, oracle">
             <column name="C_HLEVEL" valueNumeric="2"/>
             <column name="C_FULLNAME" value="\EK\i2b2INTERNAL|Demographics\i2b2INTERNAL|Demographics|Age\"/>
             <column name="C_NAME" value="Age"/>
             <column name="C_SYNONYM_CD" value="N"/>
             <column name="C_VISUALATTRIBUTES" value="FAE"/>
-            <column name="C_BASECODE" value="i2b2INTERNAL|Demographics|Age"/>
+            <column name="C_BASECODE" value="DEM|Age"/>
             <column name="C_METADATAXML"/>
-            <column name="C_FACTTABLECOLUMN" value="concept_cd"/>
-            <column name="C_TABLENAME" value="concept_dimension"/>
-            <column name="C_COLUMNNAME" value="concept_path"/>
-            <column name="C_COLUMNDATATYPE" value="T"/>
-            <column name="C_OPERATOR" value="LIKE"/>
-            <column name="C_DIMCODE" value="\Cardiovascular Registry\i2b2INTERNAL|Demographics\i2b2INTERNAL|Demographics|Age\"/>
+            <column name="C_FACTTABLECOLUMN" value="patient_num"/>
+            <column name="C_TABLENAME" value="patient_dimension"/>
+            <column name="C_COLUMNNAME" value="birth_date"/>
+            <column name="C_COLUMNDATATYPE" value="N"/>
+            <column name="C_OPERATOR" value="IS NOT"/>
+            <column name="C_DIMCODE" valueComputed="SUBSTR('NULL',1,4)"/>
```

Comment:
leaving it null is not an option since c_dimcode is not NULL field. how about 
		<column name="C_DIMCODE" value="'NULL'"/>
and then a update with a <sql> tag at the end?